src/java.base/share/classes/sun/security/ssl/HKDF.java

```diff
@@ -47,6 +65,10 @@ public final class HKDF {
     private final Mac hmacObj;
     private final int hmacLen;
 
+    /*[IF OPENJCEPLUS_SUPPORT]*/
+    private KeyGenerator hkdfGenerator;
```

Comment:
This field should be `final`, possibly initialized to `null` by the constructor.